packages/browser-destinations/destinations/rupt/src/index.ts

```diff
@@ -0,0 +1,78 @@
+import { defaultValues, DestinationDefinition } from '@segment/actions-core'
+import { browserDestination } from '@segment/browser-destination-runtime/shim'
+import type { BrowserDestinationDefinition } from '@segment/browser-destination-runtime/types'
+import attach from './attach'
+import type { Settings } from './generated-types'
+import Rupt from './types'
+
+declare global {
+  interface Window {
+    Rupt: Rupt
+  }
+}
+
+const presets: DestinationDefinition['presets'] = [
+  {
+    name: 'Attach Device',
+    subscribe: "type = 'page'",
+    partnerAction: 'attach',
+    mapping: defaultValues(attach.fields),
+    type: 'automatic'
+  }
+]
+export const destination: BrowserDestinationDefinition<Settings, Rupt> = {
+  name: 'Rupt',
+  slug: 'actions-rupt',
+  mode: 'device',
+
+  presets,
+
+  settings: {
+    client_id: {
+      description: 'The API client id of your Rupt project.',
+      label: 'Client ID',
+      type: 'string',
+      required: true
+    },
+    new_account_url: {
+      description: 'A URL to redirect the user to if they want to stop account sharing and create a new account.',
+      label: 'New Account URL',
+      type: 'string',
+      required: true
+    },
+    logout_url: {
+      description:
+        'A URL to redirect the user to if they choose to logout or if they are kicked out by a verified owner.',
```

Comment:
```suggestion
        'A URL to redirect the user to if they choose to logout or if they are kicked out by a verified owner.',
```

Can we use a friendlier term other than 'kicked out', or is that a term used by the Rupt platform/product?